specs/Transportation/CrowdFlowObserved/doc/spec.md

```diff
@@ -0,0 +1,120 @@
+# Crowd Flow Observed
+
+## Description
+
+An observation related to the movement of people at a certain place and time.
+
+## Data Model
+
++ `id`: Entity id. 
+    + It shall be `urn:ngsi-ld:CrowdFlowObserved:<identifier>` being `identifier` a unique identifier. 
+
++ `type`: Entity type. 
+    + It shall be equal to `CrowdFlowObserved`.
+    
++ `dateCreated` : Entity's creation timestamp. (`createdAt` in NGSI-LD)
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Read-Only. Automatically generated. 
+ 
++ `dateModified` : Last update timestamp of this Entity. (`modifiedAt` in NGSI-LD)
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Read-Only. Automatically generated.
+  
++ `source` : A sequence of characters giving the original source of the Entity data as a URL.
+    + Attribute type: [URL](https://schema.org/URL)
+    + Mandatory
+
++ `location` : Location of this crowd flow observation represented by a GeoJSON geometry. 
+    + Attribute type: `geo:json`. (`GeoProperty`)
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory if `refRoadSegment` is not present.
```

Comment:
wouldn't be also address ok?

Reply:
I think so, it we take address just to be for instance a street in a city